pallets/liquid-staking/src/lib.rs

```diff
@@ -1680,13 +1717,15 @@ pub mod pallet {
 
         #[require_transactional]
         pub fn do_matching() -> DispatchResult {
+            let remove_results = FastUnstakeRequests::<T>::clear(200u32, None);
```

Comment:
if clear with limit here, better to use [CountedStorageMap](https://paritytech.github.io/substrate/master/frame_support/storage/types/struct.CountedStorageMap.html)

Reply:
And also better to re-benchmark  the weight of `force_matching` based on the worst case here

Reply:
though with limit we will probably need to clear multiple times, in this case shall we set a limit for one era's unstake users?

Reply:
what I am thinking is if we use `StorageMap`, then we can store as many requests as we can in the current era, clear 200 is due to the weight limit.
In the current process, will check liquid balance when doing fast matches, so no need to clear all requests.
users' requests in the previous era can still be matched in the next era.